runtime/v2/shim/shim_nix.go

```diff
@@ -0,0 +1,102 @@
+// +build !windows
```

Comment:
I think renaming this file `shim_unix.go` without the build directive would work?